netlify/functions/getVideosList.ts

```diff
@@ -0,0 +1,37 @@
+export const getVideosList = async () => {
+  try {
+    const response = await fetch(
+      `https://video.bunnycdn.com/library/87989/videos?itemsPerPage=1000`,
```

Comment:
is 87989 our account id? Might be good to separate that out.

Reply:
It's a library ID, of which our account can hold many, but have broke it out to the env file.